helm/prometheus-meta-operator/values.yaml

```diff
@@ -7,7 +7,7 @@ Installation:
       Domain: quay.io
 pod:
   user:
-    id: 1000
+    id: 0
```

Comment:
Had to change this in order to be able to read the `key.pem` for etcd certificates, otherwise we run into `permission denied` error

```
E 08/31 15:51:14 /api/v1/namespaces/default/services/kubernetes failed to reconcile | operatorkit/v2/pkg/controller/controller.go:305 | controller=prometheus-meta-operator-control-plane-controller | event=update | loop=0 | stack=map[annotation:open /etcd-client-certs/key.pem: permission denied kind:unknown stack:[map[file:/root/project/service/controller/resource/etcd-certificates/resource.go line:99] map[file:/root/project/service/controller/resource/etcd-certificates/resource.go line:71] map[file:/root/project/service/controller/resource/generic/create.go line:14] map[file:/go/pkg/mod/github.com/giantswarm/operatorkit/v2@v2.0.0/pkg/resource/wrapper/retryresource/basic_resource.go line:52] map[file:/go/pkg/mod/github.com/giantswarm/backoff@v0.2.0/retry.go line:23] map[file:/go/pkg/mod/github.com/giantswarm/operatorkit/v2@v2.0.0/pkg/resource/wrapper/retryresource/basic_resource.go line:64] map[file:/go/pkg/mod/github.com/giantswarm/operatorkit/v2@v2.0.0/pkg/resource/wrapper/metricsresource/basic_resource.go line:43] map[file:/go/pkg/mod/github.com/giantswarm/operatorkit/v2@v2.0.0/pkg/controller/controller.go line:591] map[file:/go/pkg/mod/github.com/giantswarm/operatorkit/v2@v2.0.0/pkg/controller/controller.go line:509] map[file:/go/pkg/mod/github.com/giantswarm/operatorkit/v2@v2.0.0/pkg/controller/controller.go line:485]]] | version=10192487
```